components/backup-stream/src/router.rs

```diff
@@ -544,26 +545,44 @@ impl TempFileKey {
         }
     }
 
-    fn path_to_log_file(&self, min_ts: u64) -> String {
+    fn format_date_time(ts: u64) -> impl Display {
+        use chrono::prelude::*;
+        let millis = TimeStamp::physical(ts.into());
+        let dt = Utc.timestamp_millis(millis as _);
+        fail::fail_point!("stream_format_date_time", |s| {
+            return dt.format(&s.unwrap_or("%Y%m".to_owned())).to_string();
+        });
+        #[cfg(feature = "failpoints")]
+        return dt.format("%Y%m%d").to_string();
+        #[cfg(not(feature = "failpoints"))]
+        return dt.format("%Y%m%d");
+    }
+
+    fn path_to_log_file(&self, min_ts: u64, max_ts: u64) -> String {
         format!(
-            // "/v1/t{:012}/{:012}-{}.log",
-            "v1_t{:012}_{:012}-{}.log",
+            "t{:08}/{}-{:012}-{}.log",
```

Comment:
How to process `version` after this change? when we want to make some breaking change of data format. 🤔

Reply:
done, version added back to root dir